flexget/plugins/input/rss.py

```diff
@@ -184,6 +185,25 @@ def process_invalid_content(self, task, data, url):
             f.write(data)
         log.critical('I have saved the invalid content to %s for you to view', filepath)
 
+    def escape_content(self, content):
+        valid_escapes = ('&quot;', '&apos;', '&lt;', '&gt;', '&amp;')
```

Comment:
These (and every string literal in this function) should be changed to byte literals. i.e. `b'&quot;'`